cli/src/cmd_net.rs

```diff
@@ -692,9 +981,9 @@ pub struct CmdBgpPeerAdd {
     #[arg(long)]
```

Comment:
Line 939: form -> from?
Line 953: "to to"

Reply:
Done